### ISSUE TITLE
Add $products->setSortBy('random');

### ISSUE THREAD

**Issue by gstekelenburg** [concrete5-community-store/community_store#121]
Could this be added? I use it for myself, but can imagine others want to use this feature to :-)

`case "random":
    $query->orderBy('RAND()', null);
break;`

**Comment by Mesuva** [comment]
I don't seem why not!

The only question I have is how we actually treat 'random'.  
By doing an order by in this way it's going to randomise the order of products to a different order every page view. But would it be better to seed the random number with something that stays contant for a period of time, something like the hour of the day?   Then it would still show random, but stay in the same order while someone is browsing.

Or it could be locked to someone's session...

**Comment by gstekelenburg** [comment]
I suppose it should be locked on someone's session. Or perhaps a choice, 'couz i can imagine you want a row of 4 random products in a sidebar or homepage that changes on every page view. For the big list, it should be locked on a session/visit or perhaps date.

**Comment by Mesuva** [comment]
I've made a pull request here for two different random order sorts.
One is completely random, the other is locked daily. I figured this would be better than trying to lock it to a session - if someone rings someone up and says 'on your home page, the third product along..', if it's locked to that day everyone will see the same order.

https://github.com/concrete5-community-store/community_store/pull/133

Let me know your thoughts
